Sources/MongoSwift/ClientSession.swift

```diff
@@ -222,4 +222,52 @@ public final class ClientSession {
             }
         }
     }
+
+    /**
+     * Starts a multi-document transaction for all subsequent operations in this session. Any options provided in
```

Comment:
I think you've got all the important content in this docstring, just a wording/formatting nit: since the async methods don't actually throw, we word their docstrings a bit differently. I think we should match that here. for example, see [this docstring](https://github.com/mongodb/mongo-swift-driver/blob/ec2bba2fa2779360a427695425d7589f93e9f843/Sources/MongoSwift/MongoCollection.swift#L93) on `MongoCollection.drop`. key differences are the phrasing "if the future fails, the error is likely one of the following..." and the bulleted list of errors.

Reply:
Done.